.github/workflows/full-service-tests.yml

```diff
@@ -59,17 +59,29 @@ jobs:
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
 
+      # These steps aren't needed for unit tests
```

Comment:
Why the comment here about unit tests?

Reply:
It's a legacy comment that was added back as part of the revert. It's informational: since this workflow runs all the tests ```        gradleTask: [unitTest, connectedTest, connectedPlusTest, azureUnitTest, azureConnectedTest, azureConnectedPlusTest, awsUnitTest]
```. I've removed it to avoid confusion.